first_app/hello.py

```diff
@@ -153,12 +160,108 @@ def api_for_who_i_am(user_id):
         return {"error": "Requested data is not yours"}
 
  
-@hello_urls.route('/api/v1/<user_id>/posts/<post_id>', methods=['GET'])
-def get_user_posts(post_id):
-    """Виведення постів по користувачу. Виводимо пост користувача"""
+@hello_urls.route('/api/v1/<user_id>/posts/', methods=['GET'])
+@login_required
+def get_user_post(user_id):
+    """Список постів юзера"""
+    post = Post.query.all(user_id)
+    return "Отримали пости юзера", 200
```

Comment:
user_posts = []
for post in posts:
  _post_data = {
     "id": post.id,
}
    user_posts.append(_post_data)
return user_posts, 200